pkg/helpers/autoimport.go

```diff
@@ -163,9 +162,20 @@ func (i *ImportHelper) Import(backupRestore bool, clusterName string,
 			), false, currentRetry, nil
 	}
 
-	clientHolder, restMapper, err := i.generateClientHolderFunc(managedClusterKubeClientSecret)
+	// build import client with managed cluster kube client secret
+	result, clientHolder, restMapper, err := i.generateClientHolderFunc(managedClusterKubeClientSecret)
```

Comment:
since we removed the default `generateClientHolderFunc` in `NewImportHelper`, could this be nil?

Reply:
it could be ...., if this is nil, it must be a programmer level error, to avoid this add the check at the beginning